tests/integration/compose/docker_compose_iceberg_hms_catalog.yml

```diff
@@ -55,13 +55,13 @@ services:
     image: minio/mc
     environment:
       - AWS_ACCESS_KEY_ID=minio
-      - AWS_SECRET_ACCESS_KEY=minio123
+      - AWS_SECRET_ACCESS_KEY=ClickHouse_Minio_P@ssw0rd
       - AWS_REGION=us-east-1
     entrypoint: >
       /bin/sh -c "
-      until (/usr/bin/mc config host add minio http://minio:9000 minio minio123) do echo '...waiting...' && sleep 1; done;
-      /usr/bin/mc rm -r --force minio/warehouse;
-      /usr/bin/mc mb minio/warehouse --ignore-existing;
-      /usr/bin/mc policy set public minio/warehouse;
+      until (/usr/bin/mc config host add minio http://minio:9000 minio ClickHouse_Minio_P@ssw0rd) do echo '...waiting...' && sleep 1; done;
+      /usr/bin/mc rm -r --force minio/warehouse-hms;
+      /usr/bin/mc mb minio/warehouse-hms --ignore-existing;
```

Comment:
Maybe we should move bucket creation to the cluster.py as for other MinIOs? https://github.com/ClickHouse/ClickHouse/blob/51be0b38c944647eb6af89408965469ca9e1c828/tests/integration/helpers/cluster.py#L2607-L2659